src/data/queries/tests/__snapshots__/healthCareLocalFacility.test.tsx.snap

```diff
@@ -1,6 +1,156 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`HealthCareLocalFacility formatData outputs formatted data 1`] = `
```

Comment:
The diff is a little deceiving; I just renamed the test, so the snapshot name changed. The contents are the same, as you can see after these insertions.

Reply:
Yes! Another test is getting added. The diff is just a little deceiving in that the contents of this snapshot haven't changed, though I have changed the implementation of the test and renamed it. I just wanted to call out that the snapshot of the existing test didn't actually change. Wasn't super clear, sorry. 😅

Reply:
That's a good point about it being non-Lovell data. That's true. I'm open to renaming suggestions. 🤔